ts/features/messages/components/MessageAttachments.tsx

```diff
@@ -128,9 +136,18 @@ export const MessageAttachments = (props: Props): React.ReactElement | null =>
   props.attachments.length > 0 ? (
     <>
       {props.attachments.map((attachment, index) => (
-        <View key={index}>
+        <View
+          key={index}
+          style={
+            props.disabled
+              ? styles.attachmentContainerDisabled
+              : styles.attachmentContainerEnabled
+          }
+          testID="MessageAttachmentContainer"
+        >
```

Comment:
If we do so, the partial opacity is not applied to the `ItemSeparatorComponent` below, while in the FIGMA file even the latter is transparent.

Reply:
You are right, they are the same color. 
I moved the style to the Touchable Opacity. 

The view seems to be needed otherwise there is an `ERROR  Warning: Each child in a list should have a unique "key" prop`